content/docs/command-reference/exp/list.md

```diff
@@ -0,0 +1,44 @@
+# exp list
+
+List `dvc experiments` in a <abbr>DVC repository</abbr> (remote or local).
```

Comment:
Can `exp list` include checkpoints?

Reply:
It will include the name of a checkpoint run, but not individual commits within the checkpoint run.

Reply:
K. `show` does list checkpoints, right? (I'll try it.)

Reply:
Wait I already know it does! Per https://dvc.org/doc/api-reference/make_checkpoint#example-every-100th-iteration